conda/base/context.py

```diff
@@ -507,12 +507,26 @@ def default_channels(self):
         #   - are meant to be prepended with channel_alias
         return self.custom_multichannels[DEFAULTS_CHANNEL_NAME]
 
+    def _mac_ver(self):
+        # platform.mac_ver() returns ('', ('', '', ''), '') on Linux and Windows
+        return platform.mac_ver()[0] or '10.9'  # default to 10.9, the oldest we support
+
     @memoizedproperty
     def custom_multichannels(self):
         from ..models.channel import Channel
 
+        if self._default_channels:
+            _default_channels = self._default_channels
+        else:
+            _default_channels = list(DEFAULT_CHANNELS)
+            if (self.subdir == "osx-64" and int(self._mac_ver().split('.')[1]) < 11
+                    or self.bits == 32):
+                _default_channels.remove('https://repo.continuum.io/pkgs/mro')
+            if not on_win:
```

Comment:
Since those changes already introduce the dependency on `self.subdir`, could we also make this
```diff
-            if not on_win:
+            if self.subdir not in {"win-32", "win-64"}:
```
? While trying to "emulate" a Windows user I just noticed that I get
```
$ CONDA_SUBDIR=win-64 conda config --show default_channels
default_channels:
  - https://repo.continuum.io/pkgs/main
  - https://repo.continuum.io/pkgs/free
  - https://repo.continuum.io/pkgs/r
  - https://repo.continuum.io/pkgs/pro
```
instead of the expected
```
$ CONDA_SUBDIR=win-64 conda config --show default_channels
default_channels:
  - https://repo.continuum.io/pkgs/main
  - https://repo.continuum.io/pkgs/free
  - https://repo.continuum.io/pkgs/r
  - https://repo.continuum.io/pkgs/pro
  - https://repo.continuum.io/pkgs/msys2
```